packages/database/src/migrations/20210921051858-LESMISRepititionRatesDistrict-modifies-data.js

```diff
@@ -0,0 +1,263 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PERMISSION_GROUP = 'LESMIS Public';
+const DISTRICT_OVERLAY_GROUP_ID = '5f2c7ddc61f76a513a000215';
```

Comment:
This is what's causing the issues with your migration. This id is not guaranteed to be the same on the remote as it is on your local, so the map overlay isn't being attached to anything.

Swap this out for looking it up by name or code and it should work.